kolibri/plugins/coach/assets/src/views/reports/learner-list-page.vue

```diff
@@ -10,7 +10,7 @@
       />
       {{ pageState.contentScopeSummary.title }}
     </h1>
-    <report-subheading />
+    <report-subheading />   
```

Comment:
Let's get rid of this whitespace. It's helpful to enable the Remove Whitespace on Save setting in your editor.